WordPressAuthenticator/Unified Auth/View Related/Sign up/UnifiedSignUpViewController.swift

```diff
@@ -44,7 +46,133 @@ class UnifiedSignUpViewController: LoginViewController {
         view.backgroundColor = unifiedBackgroundColor
     }
 
+    /// Style individual ViewController status bars.
+    ///
     override var preferredStatusBarStyle: UIStatusBarStyle {
         return WordPressAuthenticator.shared.unifiedStyle?.statusBarStyle ?? WordPressAuthenticator.shared.style.statusBarStyle
     }
+
+    /// Override the title on 'submit' button
+    ///
+    override func localizePrimaryButton() {
+        submitButton?.setTitle(WordPressAuthenticator.shared.displayStrings.magicLinkButtonTitle, for: .normal)
+        submitButton?.setTitle(WordPressAuthenticator.shared.displayStrings.magicLinkButtonTitle, for: .highlighted)
+    }
+
+    /// Reload the tableview and show errors, if any.
+    ///
+    override func displayError(message: String, moveVoiceOverFocus: Bool = false) {
+        if errorMessage != message {
```

Comment:
this seems to rely on a comparison between user defined strings (error messages): just a caveat to make sure that the condition will always be satisfied when expected (e.g. are we completely sure that the `errorMessage` and `message` will always be different when the errors are different?)

Reply:
Great question! Your question prompted me to search through the existing login bugs. Since this is old code ported from an old viewController, and there is no bug reported for this problem, I'm going to deduce that it is fine the way it is.